pkgs/by-name/ei/eiffel-studio-bin/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  autoPatchelfHook,
+  fetchzip,
+  gcc,
+  gnumake,
+  gtk2-x11,
+  lib,
+  makeWrapper,
+  stdenv,
+  xorg,
+  build ? "103187",
+  libraries ? [ ],
+  platform ? "linux-x86-64",
+  version ? "19.05",
+}:
+
+stdenv.mkDerivation {
+  pname = "eiffelstudio-bin";
+  version = version;
+  src = fetchzip {
+    url = "mirror://sourceforge/eiffelstudio/EiffelStudio%20${version}/Build_${build}/Eiffel_${version}_gpl_${build}-${platform}.tar.bz2";
+    hash = "sha256-Tt4u4hydiIXB7FQnaoE/GtJt0gYRMOaVdDDUgMrDJtU=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    gtk2-x11
+    makeWrapper
+    xorg.libXtst
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+      mkdir -p $out/bin
+      cp -r . $out
+
+      substituteInPlace $out/studio/spec/${platform}/include/config.sh --replace-fail /bin/ ""
+      makeWrapper $out/studio/spec/${platform}/bin/estudio $out/bin/estudio \
+        --set ISE_EIFFEL $out \
+        --set ISE_PLATFORM ${platform} \
```

Comment:
We should probably map the platform here via an attrset or if the binary only contains one platform, we could also just use a wildcard

Reply:
see https://github.com/NixOS/nixpkgs/pull/438978#discussion_r2316063928